src/__tests__/starWarsSchema.js

```diff
@@ -164,7 +167,7 @@ const humanType = new GraphQLObjectType({
       type: new GraphQLList(characterInterface),
       description: 'The friends of the human, or an empty list if they ' +
                    'have none.',
-      resolve: human => getFriends(human),
+      resolve: (human: Human): Array<Promise<Character>> => getFriends(human),
```

Comment:
are these resolve function type hints necessary? I would expect that Flow would not require them since you already typed `getFriends`